tests/unit/test_winAPI/test_displayTracking.py

```diff
@@ -0,0 +1,81 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2022 NV Access Limited
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+
+import unittest
+
+from winAPI import displayTracking
+from winAPI.displayTracking import (
+	_updateOrientationState,
+	Orientation,
+	OrientationState,
+)
+
+
+class Test_UpdateOrientationState(unittest.TestCase):

```

Comment:
Tests to cover:
- Initial state
- No change (`lastWidth == newWidth`, `lastHeight==newHeight`)
- Width increase, landscape `(newHeight == oldHeight) < newWidth > oldWidth`
- Height increase, landscape `(newHeight > oldHeight) < newWidth == oldWidth`
- Width decrease, landscape `(newHeight == oldHeight) < newWidth < oldWidth`
- Height decrease, landscape `(newHeight < oldHeight) < newWidth == oldWidth`
- Width increase, portrait `(newHeight == oldHeight) > newWidth > oldWidth`
- Height increase, portrait `(newHeight > oldHeight) > newWidth == oldWidth`
- Width decrease, portrait `(newHeight == oldHeight) > newWidth < oldWidth`
- Height decrease, portrait `(newHeight < oldHeight) > newWidth == oldWidth`
- Orientation changed to landscape
- Orientation changed to portrait 
- Orientation changed but `newWidth == newHeight`, IE square, what should this do?